source/Module/Common/Functions.XccdfXml.ps1

```diff
@@ -283,7 +283,12 @@ function Get-PowerStigFileList
         $Path
     )
 
-    $id = Split-BenchmarkId -Id $stigDetails.Benchmark.id -FilePath $Path
+    $benchmarkId = $stigDetails.Benchmark.id
```

Comment:
can we revert these changes and just add the following after line 453. It will make the same update without impacting other STIGs

<img width="901" alt="image" src="https://user-images.githubusercontent.com/44269812/223141219-65c43720-c4f8-4514-b4e6-7eddfadb9583.png">

Reply:
Completed